db/seeds.rb

```diff
@@ -9,22 +9,25 @@ def make_books
 
 def make_users
   1.upto(5) do |n|
-    User.create!  email: "user#{n}@gmail.com",
-                  role: 0,
-                  password: "password",
-                  organization: "org#{n}"
+    user = User.create  email: "user#{n}@gmail.com",
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.